src/main/java/com/seedfinding/latticg/reversal/asm/StringParser.java

```diff
@@ -162,14 +161,22 @@ private Token consumeNumberToken() {
     public Class<?> consumeClass() {
         Token firstToken = consume();
         switch (firstToken.getText()) {
-            case "boolean": return boolean.class;
-            case "char": return char.class;
-            case "double": return double.class;
-            case "float": return float.class;
-            case "int": return int.class;
-            case "long": return long.class;
-            case "short": return short.class;
-            case "void": return void.class;
+            case "boolean":
+                return boolean.class;
+            case "char":
+                return char.class;
+            case "double":
+                return double.class;
+            case "float":
+                return float.class;
+            case "int":
+                return int.class;
+            case "long":
+                return long.class;
+            case "short":
+                return short.class;
+            case "void":
+                return void.class;
```

Comment:
This is a little verbose, is it really necessary?

Reply:
This is just the formatter putting return on next line, personally I like it but I can toggle it off if it spites you

Reply:
Yeah it's not too important